packages/ketcher-react/src/script/ui/state/functionalGroups/index.ts

```diff
@@ -42,12 +44,23 @@ const functionalGroupsReducer = (
     case 'FG_INIT':
       return { ...state, ...payload }
 
+    case 'FG_HIGHLIGHT':
+      return { ...state, functionalGroupInfo: payload }
+
     default:
       return state
   }
 }
 
 const initFGroups = (lib: SdfItem[]) => ({ type: 'FG_INIT', payload: { lib } })
+const highlightFGroup = (group: any) => ({
+  type: 'FG_HIGHLIGHT',
+  payload: group
+})
+
+export function highlightFG(dispatch, group: any) {
```

Comment:
Do we really need `dispatch` as argument? 
Can't we do
`dispatch(highlightFGroup({ group: null, id: null }))`
 instead of
`highlightFG(dispatch, { group: null, id: null })`